assets/controllers/shortcuts_controller.js

```diff
@@ -100,5 +100,16 @@ export default class extends Controller {
       e.preventDefault();
       this.showActionsTarget.click();
     });
+
+    const originalStopCallback = Mousetrap.prototype.stopCallback;
+
+    Mousetrap.prototype.stopCallback = (e, element, combo) => {
+      // allow esc key to be used in input fields of topbar
+      if (combo === 'esc' && element.dataset.topbarTarget !== undefined) {
```

Comment:
this allows the ESC key to bubble to Mousetrap when the element is a topbar target, means either the add new entry input or the search input. so that the ESC shortcut to close them and show back the top bar title works